src/app/community/_components/PostCard.tsx

```diff
@@ -22,23 +26,112 @@ interface PostCardProps {
 }
 
 export default function PostCard({ cardData, isMine }: PostCardProps) {
+  const queryClient = useQueryClient();
+
   const [isPostModalOpen, setIsPostModalOpen] = useState(false);
+  const [isPopupOpen, setIsPopupOpen] = useState(false);
+  const [isEditModalOpen, setIsEditModalOpen] = useState(false);
 
   const { id, nickName, updateAt, title, thumbnail, likeCount, commentCount, userImage, isLiked } = cardData;
 
   const ApdatedDate = new Date(updateAt);
   const timeToString = calculateTimeDifference(ApdatedDate);
 
+  const handleClickPopup = (e: MouseEvent<SVGElement>) => {
+    e.stopPropagation();
+    setIsPopupOpen(!isPopupOpen);
```

Comment:
```suggestion
    setIsPopupOpen((prevIsOpen) => !prevIsOpen);
```
작성하신 내용대로라면 이런식으로 사용하는게 더 명확할거같습니다